tests/chainer_tests/links_tests/normalization_tests/test_batch_normalization.py

```diff
@@ -243,4 +244,49 @@ def test_backward_gpu(self):
         gy = cuda.to_gpu(self.gy)
         self.check_backward(x, gy)
 
+
+class TestInitialize(unittest.TestCase):
+
+    def setUp(self):
+        self.decay = 0.9
+        self.size = 3
+        self.initial_gamma = numpy.random.uniform(-1, 1, self.size)
+        self.initial_gamma = self.initial_gamma.astype(numpy.float32)
+        self.initial_beta = numpy.random.uniform(-1, 1, self.size)
+        self.initial_beta = self.initial_beta.astype(numpy.float32)
+        self.link = links.BatchNormalization(self.size, self.decay,
+                                             initial_gamma=self.initial_gamma,
+                                             initial_beta=self.initial_beta)
+
+    @condition.retry(3)
+    def test_initialize_cpu(self):
+        assert_allclose(self.initial_gamma, self.link.gamma.data)
+        assert_allclose(self.initial_beta, self.link.beta.data)
+
+    @attr.gpu
+    @condition.retry(3)
+    def test_initialize_gpu(self):
+        self.link.to_gpu()
+        assert_allclose(self.initial_gamma, self.link.gamma.data)
+        assert_allclose(self.initial_beta, self.link.beta.data)
+
+
+class TestInitializeWithoutGammaAndBeta(unittest.TestCase):
```

Comment:
To me, the name of this class seems ambiguous because it also seems to check the behavior of `BatchNormalization` without `gamma` and `beta` attributes, although it is a matter of tastes. How about changing the name like `TestDefaultInitializer`?